Userland/Libraries/LibWeb/SVG/AttributeNames.cpp

```diff
@@ -22,6 +22,9 @@ void initialize_strings()
     ENUMERATE_SVG_ATTRIBUTES(__ENUMERATE_SVG_ATTRIBUTE)
 #undef __ENUMERATE_SVG_ATTRIBUTE
 
+    // NOTE: Special case for attributes with ':' in them.
+    xlink_href = "xlink:href"_fly_string;
```

Comment:
Hold on a minute, `xlink` is supposed to be a namespace in this attribute. We shouldn't put `xlink:` in the local name :)
Sounds like we need to be upgrading our `AttributeNames` to `QualifiedName` perhaps?